docs/deploy/2-vm.mdx

```diff
@@ -4,77 +4,113 @@ title: Deploy to a VM
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-This guide helps you to set up LiveKit server on a virtual machine on AWS EC2 or Digital Ocean.
+This guide helps you to set up production-ready LiveKit server on a virtual machine.
+
+This configuration uses Docker Compose and Caddy. Your LiveKit server will support
+a wide range of connectivity, including those behind VPN and firewalls (via TURN/TLS)
+
+You do not need separate SSL certificates for this set up, we will provision them
+automatically with Caddy.
 
 ## Pre-requisites
 
 To start, you'll need:
 
-- A valid self-generated LiveKit API key, following the instructions in the [Getting Started](/guides/getting-started#generate-api-key-and-secret) guide.
-- A valid SSL certificate issued by a [certificate authority](https://en.wikipedia.org/wiki/Certificate_authority) (self-signed certs do not work here).
-- Ability to add a DNS record pointing to your new LiveKit Server.
+- A domain that you control
+- Ability to add DNS records for subdomains that point to your new LiveKit Server
```

Comment:
The ability to add DNS records for subdomains which point to your new LiveKit server